src/XML/CombinedXmlValidationService.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace CultuurNet\UDB3\UDB2\XML;
+
+class CombinedXmlValidationService implements XMLValidationServiceInterface
```

Comment:
De naamgeving die we hiervoor doorgaans hanteren in UDB3 is "Composite".  Zie bv https://sourcemaking.com/design_patterns/composite